alcove/query/api.py

```
@@ -70,28 +208,111 @@ def search(request: Request, q: str = "", k: int = 5, collections: str = "", mod
-                "text": highlighted,
-                "source": meta.get("source", "unknown") if isinstance(meta, dict) else "unknown",
-                "collection": meta.get("collection", "default") if isinstance(meta, dict) else "default",
-                "score": round(1.0 - dist, 3) if dist <= 1.0 else round(dist, 3),
-            })
+        try:
+            raw = _dispatch_query(q, k, mode=mode, collections=coll_list)
+            documents = raw.get("documents", [[]])[0]
+            metadatas = raw.get("metadatas", [[]])[0]
+            distances = raw.get("distances", [[]])[0]
+
+            for doc, meta, dist in zip(documents, metadatas, distances):
```

Ruff (0.15.5)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)